crates/astria-composer/src/searcher/mod.rs

```diff
@@ -44,8 +50,10 @@ pub(super) struct Searcher {
     collectors: HashMap<String, Collector>,
     collector_statuses: HashMap<String, watch::Receiver<collector::Status>>,
     // A channel on which the searcher receives transactions from its collectors.
-    new_transactions: Receiver<collector::Transaction>,
-    collector_tasks: tokio_util::task::JoinMap<String, eyre::Result<()>>,
+    new_transactions_rx: Receiver<collector::Transaction>,
+    new_transactions_tx: Sender<collector::Transaction>,
+    rollups: HashMap<String, String>,
```

Comment:
can you add a comment explaining what these fields are?

Reply:
Added